license_manager/apps/api/tasks.py

```
@@ -174,6 +180,11 @@ def license_expiration_task(license_uuids):
     try:
         enterprise_api_client = EnterpriseApiClient()
         enterprise_api_client.bulk_licensed_enrollments_expiration(expired_license_uuids=license_uuids)
+        logger.info(
```

Should we re-raise this exception that is caught here so callers know there was an issue expiring the enrollments?